react/features/mobile/navigation/components/welcome/components/WelcomePageNavigationContainer.js

```diff
@@ -4,21 +4,21 @@ import { createDrawerNavigator } from '@react-navigation/drawer';
 import React from 'react';
 import { useTranslation } from 'react-i18next';
 
+import WelcomePage from '../../../../../welcome/components/WelcomePage';
+import { drawerContentOptions } from '../../../../../welcome/components/constants';
+import HelpView from '../../../../../welcome/components/native/help/components/HelpView';
```

Comment:
Can we have all these components in a welcome/components/native directory instead? It feels a bit too much to have a separate folder for each screen IMHO.

Reply:
Done.